README.markdown

```diff
@@ -218,6 +218,7 @@ in your view_templates:
 <%= yield(:article_subtitle) %>
 <%= yield(:article_teaser) %>
 ```
+!!!Attention!!! The layouts' styling will only work if you include `gem bootstrap-sass` in Your Gemfile
```

Comment:
[typo] Your -> your and `.` at the end of the sentence

And I would add the link to the gem, as well as setting the !!! Attention !!! as a smaller headline to be more visible for readers.

- [ ] link to the gem


```suggestion
#### !!!Attention!!!

The layouts' styling will only work if you include `gem bootstrap-sass` (https://github.com/twbs/bootstrap-sass) in your Gemfile.
```

- [ ] make smaller headline
- [ ] `.` at the end of the sentence

**Before**:
<img width="948" alt="bildschirmfoto 2019-01-28 um 15 46 02" src="https://user-images.githubusercontent.com/1942953/51843842-22b33a00-2314-11e9-9728-ef66b5db3364.png">

**After**:
<img width="914" alt="bildschirmfoto 2019-01-28 um 15 50 08" src="https://user-images.githubusercontent.com/1942953/51843971-6c9c2000-2314-11e9-93b4-b75e96c08657.png">